### PR TITLE
Trello

### PR DESCRIPTION
**Pull request by Joanb2**

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ $(function() {
         		self.removeColumn();
 			});
 		    $columnAddCard.on('click', function(event) {
-        		self.addCard(new Card(prompt("Enter the name of the card")));
+        		self.addCard(new Card());
```
Comment (matwjablonski) [inline js/scripts.js]: Card() is without any argument, why?

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: js/scripts.js [hunk 2]
```diff
@@ -48,6 +49,15 @@ $(function() {
 	    }
 	};
 
+	function Input() {
```
Comment (matwjablonski) [inline js/scripts.js]: this is simple function - so name of this functions should start with small character

File: js/scripts.js [hunk 3]
```diff
@@ -48,6 +49,15 @@ $(function() {
...
+	 var $cardInput = $('#card-input').val();
```
Comment (matwjablonski) [inline js/scripts.js]: indentation

File: index.html [hunk 4]
```diff
@@ -6,6 +6,10 @@
     <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+    <form>
+        <label>Card:</label>
+        <input type="text" id ="card-input" title="tooltip" value=" ">
```
Comment (matwjablonski) [inline index.html]: what will happen when user click Enter? ;)

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: index.html (hunk 4 above)
Comment (Joanb2) [reply index.html]: Nothing will happend :) The value of the input is not direct to any of the column at this moment.

Comment (matwjablonski) [reply index.html]: your page will reload ;) prevent this

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**